certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
```

Comment:
That's fair. If you wanted we could also change the loop from `while True:` to `for _ in range(n)` where `n>=100` and error out after the loop.